resilience4j-framework-common/src/main/java/io/github/resilience4j/common/circuitbreaker/configuration/CommonCircuitBreakerConfigurationProperties.java

```diff
@@ -696,6 +696,10 @@ public Double getExponentialBackoffMultiplier() {
 
         public InstanceProperties setExponentialBackoffMultiplier(
             Double exponentialBackoffMultiplier) {
+            if (exponentialBackoffMultiplier <= 0) {
```

Comment:
I've added this one to avoid negative values of `intervalBiFunction` and thus negative Durations